src/test/java/jUnit/JsonParserTest.java

```diff
@@ -12,27 +11,35 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import static jUnit.Utils.Constants.FILE_NAME_LIST;
 import static org.junit.jupiter.api.Assertions.*;
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class JsonParserTest {
 
-    static Cart initialCart;
-    static JsonParser parser;
+    Cart initialCart;
+    JsonParser parser;
+
+    List<String> fileNameList = Stream.of(
```

Comment:
as agreed, let's create and use single file name (String with two specifiers for name and extension) for read-write test cases. There is no benefit in using list of almost identical file names